changes/2020-06-16_required-examples/change.md

```diff
@@ -0,0 +1,98 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Define Required Examples
+
+## Affected Specifications
+
+This serves as a reference of all specification documents that this change affects.
+
+| Specification                                         |
+| ----------------------------------------------------- |
+| [examples](../../examples/examples.md)                |
+| [examples readme](../../examples/templates/readme.md) |
+
+## Affected Implementations
+
+This serves as a reference for all implementations that this change affects.
+
+| Language   | Repository                                                                            |
+| ---------- | ------------------------------------------------------------------------------------- |
+| Python     | [aws-encryption-sdk-c](https://github.com/aws/aws-encryption-sdk-python)              |
+| Java       | [aws-encryption-sdk-javascript](https://github.com/aws/aws-encryption-sdk-java)       |
+| C          | [aws-encryption-sdk-c](https://github.com/aws/aws-encryption-sdk-c)                   |
+| Javascript | [aws-encryption-sdk-javascript](https://github.com/aws/aws-encryption-sdk-javascript) |
+
+## Definitions
+
+### Conventions used in this document
+
+The key words
+"MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in
+[RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Summary
+
+This document defines the examples that each implementation MUST include.
+
+## Out of Scope
+
+The specific implementations details for each example
+are out of scope for this document.
+The template document for each example contains those details.
+
+## Motivation
+
+The two primary motivations for requiring examples are
+to provide guidance to AWS Encryption SDK customers
+and to force AWS Encryption SDK developers
+to use the APIs that they design and build
+to verify that they provide the intended user experience.
```

Comment:
This is the same thing,
but I also point out that
this helps us work backwards
from the customers wanting to use the tool.

If our process of development includes
generating the examples
we are more likely to build something
that customers will successfully use.

Maybe this is just here for posterity,
but it seems like an additional nice feature
of having examples.